### ISSUE TITLE
[Bug] No Haptic feedback from Home Button

### ISSUE THREAD

**Issue by 0utlawActual** [ryannair05/Little11#11]
With Little11 installed, the Home Button does not provide any Haptic Feedback.

iPhone 7+
IOS 13.3.1
Checkra1n
Little11 1.2.0

**Comment by ryannair05** [comment]
As shown by issue #6 this is an issue with Cercube, and can be fixed by going to Filza, navigating to /Library/MovieSubstrate/DynamicLibrares/ opening CercubeServices.plist, removing com.apple.backboardd from the Bundles, and then respringing to fix the conflict with Cercube.

**Comment by 0utlawActual** [comment]
Assuming */MovieSubstarte/* was just a typo for MobileSubstrate, following these instructions had no positive effect on HomeButton Haptic Feedback. I really wanted it to work.

**Comment by ryannair05** [comment]
What other tweaks do you have?

**Comment by 0utlawActual** [comment]
It's quite a long list. I appreciate you looking into this.

https://pastebin.com/NwxMHzTS

**Comment by 0utlawActual** [comment]
So using iCleanerPro to disable/enable tweaks, I was able to isolate the conflicting tweaks. You were correct that Cercube was conflicting, but it was not the only one and that is why when I removed the com.apple.backboardd Bundel from CercubeServices.plist, the issue did not resolve. The second tweak causing the issue is Gyration. I use Gyration to enable screen rotation on Home Screen because Little11 disables it. 

After removing  com.apple.backboardd Bundle from CercubeServices.plist as well as Gyration.plist located in /Library/MobileSubstrate/DynamicLibrares/, I now have regained Haptic Feedback from HomeButton, can take screenshots, screen rotation still works with Gyration and Cercube seems to be doing it's thing as well. 

Thanks for you assistance
